### PR TITLE
Jeff

### PR DESCRIPTION
**Pull request by Jeffotter**

### CODE REVIEW

File: docs/6_jupyterhub.md [hunk 1]
```diff
@@ -43,95 +43,149 @@ The typical development workflow is to use JupyterHub and more specifically the
...
 ## Terminal
 -------------------------
 ![](../assets/images/terminal.png)
 
-You can launch a terminal from JupyterHub.  This provides access to commands you would normally run on the command line on the Yens, 
+You can launch a terminal from JupyterHub. This provides access to commands you would normally run on the command line on the Yens, 
 but through the web browser. However, we have seen issues with JupyterHub terminal (as well as VSCode) that modifies or overwrites python paths and environment variables so for package installations, we recommend using a terminal outside of JupyterHub.
 
-Let's open up a Terminal and navigate to the rf_bootcamp_2024
+Let's open up a Terminal and navigate to the `rf_bootcamp_2024`
```
Comment (natalya-patrikeeva) [inline docs/6_jupyterhub.md]: missed one!

File: docs/6_jupyterhub.md [hunk 2]
```diff
@@ -168,7 +223,7 @@ If your processes require more than these limits, reach out to the <a href="http
 Finally, you can also edit text files like R scripts directly on JupyterHub. Clicking on Text File icon will open a new file that you can edit. Similarly, clicking on Python File will create an empty `.py` file and clicking on R File will create an empty `.r` file.
 You can also navigate to a directory that has the scripts you want to edit and double click on the script name to open it up in the Text Editor.
 
-For example, navigate to `rf_bootcamp_2024/examples/r_examples` folder in file brower first then double click on `investment-npv-parallel.R` file to open it in the text editor:
+For example, navigate to `intro_to_yens/examples/` folder in file brower first then double click on `investment-npv-parallel.R` file to open it in the text editor:
```
Comment (natalya-patrikeeva) [inline docs/6_jupyterhub.md]: the git repo is "intro_to_yens_2024"

**Review by natalya-patrikeeva — COMMENTED** [review]
minor folder name changes needed

File: docs/6_jupyterhub.md (hunk 2 above)
Comment (Jeffotter) [reply docs/6_jupyterhub.md]: Oh shoot I dont think the change pushed my apologies.

**Pull request merged**